sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerConfig.java

```diff
@@ -42,6 +50,10 @@
   private static final String USER_AGENT_PREFIX = "Apache_Beam_Java";
   // A default host name for batch traffic.
   private static final String DEFAULT_HOST = "https://batch-spanner.googleapis.com/";
+  // Deadline for Commit API call.
+  private static final Duration DEFAULT_COMMIT_DEADLINE = Duration.standardSeconds(15);
```

Comment:
Can this be made configurable.

Reply:
Configurable using SpannerIO.Write.withCommitDeadline()
as is DEFAULT_MAX_CUMULATIVE_BACKOFF with SpannerIO.Write.withMaxCumulativeBackoff()